Kickstarter-iOS/Views/Controllers/LoginViewController.swift

```diff
@@ -132,8 +132,16 @@ internal final class LoginViewController: UIViewController {
       }
 
     self.viewModel.outputs.logIntoEnvironment
+      .observeForControllerAction()
```

Comment:
I added this bit because the compiler was throwing a fatal error about the main thread. I wasn't sure if `observeForUI()` was the right method to plug into here and chose `observeForControllerAction()` after looking at the same paradigm elsewhere in the view controller.

Reply:
Yeah whenever we're presenting or pushing we use `observeForControllerAction()` to be sure. I think if you click through on those functions there should be a comment about why they're there.